components/cdc/src/service.rs

```diff
@@ -128,6 +132,15 @@ impl EventBatcher {
                     self.buffer.push(ChangeDataEvent::default());
                 }
                 self.last_size += size;
+
+                let event = e.event.as_ref().unwrap();
+                match event {
+                    Event_oneof_event::Error(err) => {
+                        info!("sending error"; "error" => ?err);
```

Comment:
```suggestion
                        info!("cdc sending error"; "error" => ?err);
```

Could you also log event's regoin ID?